react-native/react/actions/search.js

```diff
@@ -46,6 +46,7 @@ const submitSearch_debounced = _.debounce((base, term, dispatch, getState) => {
   const bad_nonce = () => (getState().search.getIn([base, 'nonce']) !== nonce)
 
   const doRPC = (...args) => new Promise((resolve, reject) => {
+    // TODO think about using rpc2
     engine.rpc(...args, (error, results) => {
```

Comment:
this is tough since its abstracted in a strange way, i think we likely don't need this kind of construct anymore?